fiona/schema.pyx

```diff
@@ -0,0 +1,83 @@
+from six import text_type
+
+from fiona.errors import SchemaError
+from fiona.rfc3339 import FionaDateType, FionaDateTimeType, FionaTimeType
+
+
+cdef extern from "gdal.h":
+    char * GDALVersionInfo (char *pszRequest)
+
+
+def get_gdal_version_num():
```

Comment:
I'm not keen on the duplication of this from `ogrext.pyx` although I can see why it's needed. Can we either `cdef` it or prefix an `_` so that users aren't tempted to do `from fiona.schema import get_gdal_version_num`?